js/core/utils/type.js

```diff
@@ -74,7 +74,7 @@ const isWindow = function(object) {
 };
 
 const isRenderer = function(object) {
-    return !!(object.jquery || object.dxRenderer);
+    return object && !!(object.jquery || object.dxRenderer);
```

Comment:
The method should return `boolean` but in case the `object` is `null`, it will not be custed.
```suggestion
    return !!(object?.jquery || object?.dxRenderer);
```